lib/Admin.php

```diff
@@ -607,10 +607,17 @@ public function add_image_to_library(): void {
 	public static function get_widen_url_path( $url ) {
 		$url_arr  = wp_parse_url( $url );
 		$url_path = $url_arr['path'];
+		$path     = '';
 
-		$path = ltrim( $url_path, '/wp-content/uploads/' );
+		// Remove the WordPress uploads base path from the URL.
+		if ( is_multisite() ) {
+			$blog_id = get_current_blog_id();
+			$path    = preg_replace( "/^\/wp-content\/uploads\/sites\/$blog_id\//", '', $url_path );
+		} else {
+			$path = preg_replace( '/^\/wp-content\/uploads\//', '', $url_path );
+		}
 
-		return $path;
+		return ltrim( $path, '/' );
```

Comment:
Using ltrim here to remove the leading slash before saving the attachment to the database. WordPress core removes the leading slash for traditionally uploaded images so this was added for consistency within attachment data.